.github/actions/deploy-backend/action.yml

```diff
@@ -333,7 +332,10 @@ runs:
 
     - name: Validate function app checksum
       if: inputs.checksum-validation == 'true'
+
       uses: JosiahSiegel/checksum-validate-action@ebdf8c12c00912d18de93c483b935d51582f9236
+      ## emvaldes@hotmail.com (Replace) uses: ./.github/actions/checksum-validate-action
```

Comment:
The objective here is to provide reference to where we need to change repos that are referenced within the project.

Reply:
I have updated the email address to reflect the Aquia corporate email.